src/utils/azure.ts

```diff
@@ -26,35 +30,77 @@ export const agid: IAzureConfig = {
   cosmosNotificationStatusContainer: "notification-status",
   cosmosProfilesContainer: "profiles",
   cosmosServicesContainer: "services",
+  cosmosSenderServicesContainer: "sender-services",
+  cosmosUserDataProcessingContainer: "user-data-processing",
+  cosmosUserBonusesContainer: "user-bonuses",
   cosmosName: "agid-cosmosdb-test",
   resourceGroup: "agid-rg-test",
   storageMessagesContainer: "message-content",
+  storageBonusRedeemedContainer: "",
   storageName: "agidstoragetest"
 };
 
 export const dev: IAzureConfig = {
-  configName: "io-dev-aks-k8s-01",
-  cosmosDatabaseName: "io-dev-sqldb-db-01",
+  configName: "io-d",
+  cosmosDatabaseName: "raimondo",
```

Comment:
what? :)

Reply:
We finished free cosmos containers :) Anyway I fixed it with standard `db` container.
I think that `configName` is never used.